lib/analyzer/binary-version-extractors/openjdk-jre.ts

```diff
@@ -46,3 +43,31 @@ function parseOpenJDKBinary(fullVersionOutput: string) {
     version,
   };
 }
+
+const pkgManagerPkgNames = {
+  Apt: [/openjdk-\d*-jre-headless/,
```

Comment:
Maybe give an example for every regex to which package it conforms in a comment?